docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -278,65 +280,117 @@ Events:  <none>
-99s         Normal    SuccessfulCreate            job/sample-domain1-introspector         Created pod: sample-domain1-introspector-fqzjv
-98s         Normal    Created                     pod/sample-domain1-introspector-fqzjv   Created container sample-domain1-introspector
-98s         Normal    Pulled                      pod/sample-domain1-introspector-fqzjv   Container image "domain-home-in-image:12.2.1.4" already present on machine
-98s         Normal    Started                     pod/sample-domain1-introspector-fqzjv   Started container sample-domain1-introspector
-78s         Normal    Scheduled                   pod/sample-domain1-admin-server         Successfully assigned sample-domain1-ns/sample-domain1-admin-server to doxiao-1
-77s         Normal    Created                     pod/sample-domain1-admin-server         Created container weblogic-server
-77s         Normal    Started                     pod/sample-domain1-admin-server         Started container weblogic-server
-77s         Normal    Pulled                      pod/sample-domain1-admin-server         Container image "domain-home-in-image:12.2.1.4" already present on machine
-45s         Normal    DomainProcessingCompleted   domain/sample-domain1                   Successfully completed processing domain resource sample-domain1
-45s         Normal    Scheduled                   pod/sample-domain1-managed-server2      Successfully assigned sample-domain1-ns/sample-domain1-managed-server2 to doxiao-1
-45s         Normal    Scheduled                   pod/sample-domain1-managed-server1      Successfully assigned sample-domain1-ns/sample-domain1-managed-server1 to doxiao-1
-44s         Normal    Started                     pod/sample-domain1-managed-server2      Started container weblogic-server
-44s         Normal    Started                     pod/sample-domain1-managed-server1      Started container weblogic-server
-44s         Normal    Created                     pod/sample-domain1-managed-server2      Created container weblogic-server
-44s         Normal    Pulled                      pod/sample-domain1-managed-server2      Container image "domain-home-in-image:12.2.1.4" already present on machine
-44s         Normal    Pulled                      pod/sample-domain1-managed-server1      Container image "domain-home-in-image:12.2.1.4" already present on machine
-44s         Normal    Created                     pod/sample-domain1-managed-server1      Created container weblogic-server
-
-```
+sample-domain1-ns             13m         Normal    NamespaceWatchingStarted    namespace/sample-domain1-ns               Started watching namespace sample-domain1-ns
+kube-system                   13m         Normal    Pulling                     pod/coredns-5d4dd4b4db-qxs7l              Pulling image "k8s.gcr.io/coredns:1.3.1"
```

Comment:
Should we remove the coredns events from here?

Reply:
yes. removed.